cmd/pg/backup_fetch.go

```diff
@@ -16,18 +19,26 @@ For information about pattern syntax view: https://golang.org/pkg/path/filepath/
 	restoreSpecDescription        = "Path to file containing tablespace restore specification"
 	reverseDeltaUnpackDescription = "Unpack delta backups in reverse order (beta feature)"
 	skipRedundantTarsDescription  = "Skip tars with no useful data (requires reverse delta unpack)"
+	targetUserDataDescription     = "Fetch storage backup which has the specified UserData"
 )
 
 var fileMask string
 var restoreSpec string
 var reverseDeltaUnpack bool
 var skipRedundantTars bool
+var targetUserData string
 
 var backupFetchCmd = &cobra.Command{
-	Use:   "backup-fetch destination_directory backup_name",
+	Use:   "backup-fetch destination_directory [backup_name OR target-* flag]",
```

Comment:
Why not like this ? 
[backup_name] [--target-user-data \<data\>]

Reply:
I tried to make it clear that only one of these should be specified

Reply:
maybe [backup_name | --target-user-data <data>] ?

Reply:
Fixed in a537a95